Hazelnut/imgui.ini

```diff
@@ -21,7 +21,7 @@ Collapsed=0
 DockId=0x00000002,0
 
 [Docking][Data]
-DockSpace   ID=0x3BC79352 Window=0x4647B76E Pos=320,351 Size=1280,701 Split=X Selected=0x995B0CF8
+DockSpace   ID=0x3BC79352 Window=0x4647B76E Pos=190,232 Size=1280,701 Split=X Selected=0x995B0CF8
```

Comment:
Do not commit any `imgui.ini` changes

Reply:
Should I fix the commit on my end or is there a way to discard specific file changes on your end? I've never had to manage pull requests myself so this is more out of curiosity for me.

Also, shouldn't `imgui.ini` be gitignored completely? Or does it need to be there for the Hazelnut UI to show up properly for other users?

Reply:
> Should I fix the commit on my end or is there a way to discard specific file changes on your end? I've never had to manage pull requests myself so this is more out of curiosity for me.

There are several ways of resetting it. You can reset the file to a specific commit, or you could just commit the change to it manually such that the old state in the repo is equal to the new state in this PR. As I'm a collaborator I can also push commits to your branch, but I prefer not to (only minor details before committing). If I commit changes it will appear just the same as you did it, in a commit on your branch. That commit will then update this PR as it does for any other commit.

> Also, shouldn't `imgui.ini` be gitignored completely? Or does it need to be there for the Hazelnut UI to show up properly for other users?

There were numerous issues/PRs already to add it to gitignore. I refer to https://github.com/TheCherno/Hazel/pull/204#issuecomment-583718303 for the reason why it's not in gitignore. In the future we should add some system in place to load preferences, and at that point we should provide (manually) a default, if it doesn't exist.